store/copr/mpp.go

```diff
@@ -500,7 +500,7 @@ func (c *MPPClient) DispatchMPPTasks(ctx context.Context, variables interface{},
 		finishCh:                   make(chan struct{}),
 		cancelFunc:                 cancelFunc,
 		respChan:                   make(chan *mppResponse, 4096),
-		startTs:                    dispatchReqs[0].StartTs,
+		startTs:                    startTs,
```

Comment:
I think if `dispatchReqs` is empty, we don't have to set startTs? Then we can save an parameter.

Reply:
it will return this mppIters to mppGather, and I'm not sure if there are more behavoir related to this..